### ISSUE TITLE
Cake Console

### ISSUE THREAD

**Issue by nadiaks** [beeman/cakephp-bootstrap-notifications#3]
Hi!

For some reason when I try to load the schema.php I am getting the following error.

The chosen schema could not be loaded. Attempted to load:
File: /my/path/to/app/Plugin/notifications/Config/Schema/schema.php
Name: Notification

I am not sure if it is something I have installed incorrect or with the actual schema.php file. Any help with be appreciated! 

Thanks! :)

**Comment by Rezguino** [comment]
Hi, I have the same error if someone could help us! 
thanks

**Comment by pschiel** [comment]
The plugin uses a plural form in the classname ("Notifications"), which is against CakePHP conventions, so this part does't work out of the box. (notice the error message "Name: Notification" looking for the singular name).

workaround is providing additional class name to the command, e.g.:
"cake schema create -p Notifications Notifications"

**Comment by Rezguino** [comment]
Yeah thank you i have fixed this bug already (with some help in cakephp irc channel) and i forget to update this post. 
But it will be nice if you update the documentation. I have really bad time implementing this plugin like the notification icon don't show up, i can't see any notification just the loading! ( i consulted the source code and i found in  View / Elements / NotificationIcon.ctp only the implementation of the loading.gif ( as i can see there's no implementation of  notification information ). 
Thank you.

**Comment by beeman** [comment]
Hi guys, thanks for the comments and the suggested fixes. 

I ended up not using this plugin myself but will try to update it soon :-)
